css/components.css

```diff
@@ -112,12 +112,15 @@
 }
 
 .treeview label:hover x-line-wrap, .treeview label.hover x-line-wrap{
-    background: var(--selected-color2);
+    background: var(--under-cursor-color) !important;
```

Comment:
This and the !important above are because of the conflict when an element is already selected, and it is being hovered over as well. These two states are competing for a background color. While testing this locally, I found that the UX was not showing the hover highlight. The !important fixes this.

Of course, if you think the selected state should take precedence over the hover'd state, then I can remove the !important.

Reply:
`!important` is sort of an hack and should be avoided if possible. In this case you can probable change the css priority order:
   Move the definition of hover below/after the definition of `.selected`
if that doesn't work
   Make the definition more granular
```
     .treeview label:hover x-line-wrap,
     .treeview label.hover x-line-wrap,
     .treeview label.selected:hover x-line-wrap,
     .treeview label.selected.hover x-line-wrap { ... }
```

similar permutation of `last_selected`

Reply:
Ahhh, that makes sense. I will update to this.